progressive-image-element.js

```diff
@@ -1,11 +1,17 @@
+const LAZY_LOADING_SUPPORT = 'loading' in HTMLImageElement.prototype;
+
 class ProgressiveImageElement extends HTMLElement {
   constructor() {
     super();
     this._placeholderImage = this.querySelector('img');
     this._image = this._placeholderImage?.cloneNode(true) || new Image();
+    if (LAZY_LOADING_SUPPORT) return;
```

Comment:
You might also need to see if there's a width / height set because Chrome at least warns about these not being set. Haven't dug deep enough about `loading="lazy"` out in the wild but I've seen that notice a bunch.

Reply:
I was thinking about this as well. I'd tackle this in a separate [PR](https://github.com/andreruffert/progressive-image-element/pull/4).